tests/h/util/markdown_render_test.py

```diff
@@ -6,15 +6,15 @@
 class TestRender:
     def test_it_renders_markdown(self):
         actual = markdown_render.render("_emphasis_ **bold**")
-        assert actual == "<p><em>emphasis</em> <strong>bold</strong></p>\n"
+        assert actual == "<p><em>emphasis</em> <strong>bold</strong></p>"
```

Comment:
Seems `markdown` just doesn't put a newline on the end... which seems fine? :shrug: I don't think we care about this.

Reply:
Yeah, don't care